src/reducers/__tests__/index.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import rootReducer from '../index';
+
+describe('rootReducer', () => {
```

Comment:
A recommended way to test the rootReducer should be this [way](https://github.com/reduxjs/redux/issues/1412).

Reply:
I have made the changes